adhocracy-plus/config/settings/base.py

```diff
@@ -14,10 +14,6 @@
 # General settings
 CONTACT_EMAIL = "contact@domain"
 
-# Link to a dokuwiki instance containing a manual for aplus
-# Leave blank to disable
-APLUS_MANUAL_URL = ""
-
```

Comment:
Maybe for more transparency, better to comment this part out, and indicate that the link is set in the wagtail @shn-liqd